src/main/java/meteordevelopment/meteorclient/utils/player/ChatUtils.java

```diff
@@ -57,6 +57,14 @@ public static void registerCustomPrefix(String packageName, Supplier<Text> suppl
     public static void unregisterCustomPrefix(String packageName) {
         customPrefixes.removeIf(pair -> pair.getLeft().equals(packageName));
     }
+    
+    public static Pair<String, Supplier<Text>> getCustomPrefix(String packageName) {
+        for (Pair<String, Supplier<Text>> pair : customPrefixes) {
+            if (pair.getLeft().equals(packageName)) {
+                return pair;
```

Comment:
Is there a need to return the pair directly? The left side of the pair is just the same package name that you input to the method

Reply:
no, your correct mb